### PR TITLE
doc: concat_idents! macro: more on its limitations.

### PR DESCRIPTION
**Pull request by nodakai**
Just a small documentation change.

It would be great if anyone could check my English.

### CODE REVIEW

File: src/libstd/macros.rs [hunk 1]
```diff
@@ -269,9 +269,10 @@ pub mod builtin {
     /// This macro takes any number of comma-separated identifiers, and
     /// concatenates them all into one, yielding an expression which is a new
     /// identifier. Note that hygiene makes it such that this macro cannot
-    /// capture local variables, and macros are only allowed in item,
-    /// statement or expression position, meaning this macro may be difficult to
-    /// use in some situations.
+    /// capture local variables.  Also, as a general rule, macros are only
```
Comment (steveklabnik) [inline src/libstd/macros.rs]: one space after a period please!

File: src/libstd/macros.rs [hunk 2]
```diff
@@ -269,9 +269,10 @@ pub mod builtin {
...
+    /// allowed in item, statement or expression position.  That means while
```
Comment (steveklabnik) [inline src/libstd/macros.rs]: and here

**Comment by steveklabnik** [comment]
Your English is great! Just a small nit about the spaces and let's :shipit:

**Comment by nodakai** [comment]
Thanks, corrected to a single space style.

**Pull request merged**